File: eFenkaFrontend/src/app/models/person/person.ts
import {Language} from '../../enums/language/language.enum';

export class Person {
  id: number;
  firstname: string;
  lastname: string;
  birthDate: Date;
  language: Language;
  driversLicenseNumber: string;
  driversLicenseType: string;
  startDateDriversLicense: Date;
  endDateDriversLicense: Date;
  gender: string;
  title: string;
  picture: Int8Array;

  constructor(id: number, firstname: string, lastname: string, birthDate: Date, language: Language,
              driversLicenseNumber: string, driversLicenseType: string, startDateDriversLicense: Date, endDateDriversLicense: Date,
              gender: string, title: string) {
    this.id = id;
    this.firstname = firstname;
    this.lastname = lastname;
    this.birthDate = birthDate;
    this.language = language;
    this.driversLicenseNumber = driversLicenseNumber;
    this.driversLicenseType = driversLicenseType;
    this.startDateDriversLicense = startDateDriversLicense;
    this.endDateDriversLicense = endDateDriversLicense;
    this.gender = gender;
    this.title = title;
  }

}

File: BackEnd/eMenka/eMenka.Domain/Classes/Person.cs
﻿using System;
using eMenka.Domain.Enums;

namespace eMenka.Domain.Classes
{
    public class Person
    {
        public int Id { get; set; }
        public string Firstname { get; set; }
        public string Lastname { get; set; }
        public DateTime? BirthDate { get; set; }
        public Language Language { get; set; }
        public string DriversLicenseNumber { get; set; }
        public string DriversLicenseType { get; set; }
        public DateTime? StartDateDriversLicense { get; set; }
        public DateTime? EndDateDriversLicense { get; set; }
        public string Gender { get; set; }
        public string Title { get; set; }
        public byte[] Picture { get; set; }
    }
}